dxvk.conf

```diff
@@ -312,6 +312,18 @@
 # d3d11.exposeDriverCommandLists = True
 
 
+# Reproducible Command Stream
+#
+# Ensure that for the same D3D commands the output VK commands
+# don't change between runs. Useful for comparative benchmarking,
+# can negatively affect performance.
+#
+# Supported values:
+# - True/False
+
+# d3d11.reproducibleCommandStream = False
```

Comment:
Since both the d3d9 and d3d11 option does the same and have the same description it is fine to just list both in one place

```

# d3d11.reproducibleCommandStream = False
# d3d9.reproducibleCommandStream = False

```

Reply:
Makes sense